qemu/tests/rh_kernel_update.py

```diff
@@ -42,7 +42,7 @@ def get_brew_url(mnt_path, download_root):
         logging.debug("Brew URL is %s" % url)
         return url
 
-    def install_rpm(session, url, upgrade=False, nodeps=False, timeout=600):
+    def install_rpm(session, url, upgrade=False, nodeps=False, timeout=3600):
```

Comment:
IMO, the default value = 3600 will be overwritten if there is another value passed.
But @luckyh  thinks the caller should ensure that, then it's acceptable.